samples/Samples.Server/StartupWithRouting.cs

```diff
@@ -47,7 +48,7 @@ public void ConfigureServices(IServiceCollection services)
             .AddSystemTextJson()
             .AddErrorInfoProvider<CustomErrorInfoProvider>()
             .AddDataLoader()
-            .AddUserContextBuilder(context => new Dictionary<string, object> { { "user", context.User.Identity.IsAuthenticated ? context.User : null } })
+            .AddUserContextBuilder(context => new Dictionary<string, object?> { { "user", context.User.Identity!.IsAuthenticated ? context.User : null } })
```

Comment:
Also check README.md for API changes and other stuff.

Reply:
I'm planning to rewrite the readme, subject to your review.